source/TextBlade.ConsoleRunner/NewGameRunner.cs

```diff
@@ -9,19 +9,20 @@ namespace TextBlade.ConsoleRunner;
 /// <summary>
 /// Terribly named; the thing that does all your new-game stuff.
 /// </summary>
-public class NewGameRunner
+public class NewGameRunner(Game game)
```

Comment:
Very cool, I didn't know you could do this...

Reply:
That's new in c# 12. It's actually a mistake: rider keeps suggesting this to me and I think I let it refactor this by error. It works though :)